locales/fr_FR/pcsx2_Main.po

```diff
@@ -1720,16 +1718,16 @@ msgid ""
 msgstr "Modifie les paramètres d'émulation hardware gérés par PCSX2."
 
 #: pcsx2/gui/MainFrame.cpp:785
-msgid "Window Settings..."
-msgstr "Paramètres du mode fenêtré..."
+msgid "&Window Settings..."
+msgstr "Paramètres du mode &fenêtré..."
 
 #: pcsx2/gui/MainFrame.cpp:786
 msgid "Modify window and appearance options, including aspect ratio."
 msgstr "Modifie la fenêtre et l'apparence dont le ratio d'aspect (4:3, 16:9)."
```

Comment:
You could remove "(4:3, 16:9)" here. It's not in the original message and doesn't include all options(stretch, which could be any aspect ratio).